src/command_modules/azure-cli-cloud/azure/cli/command_modules/cloud/custom.py

```diff
@@ -39,6 +39,8 @@ def _build_cloud(cloud_name, cloud_config=None, cloud_args=None):
         cloud_args = cloud_config
     c = Cloud(cloud_name)
     for arg in cloud_args:
+        if arg == 'profile' and cloud_args[arg] is not None:
+            c.profile = cloud_args[arg]
```

Comment:
Saw this pattern been used a few times. Can this `if` branch be moved out of the loop and become
```python
   c.profile = cloud_args.get('profile', None)
```